lib/commons/text/native-element-type.js

```diff
@@ -1,3 +1,4 @@
+// @deprecated use the standards object instead
```

Comment:
This needs its own PR, so we can properly include a deprecation notice in the commit message.